charts/openshift-metering/templates/reporting-operator/reporting-operator-deployment.yaml

```diff
@@ -447,12 +462,12 @@ spec:
 {{- end }}{{/* end of authProxy.enabled */}}
       volumes:
 {{- if and .Values.networking.useGlobalProxyNetworking .Values.networking.proxy.config.trusted_ca_bundle }}
-     - name: reporting-operator-trusted-ca-bundle
-       configMap:
-         name: reporting-operator-trusted-ca-bundle
-         items:
-         - key: ca-bundle.crt
-           path: ca-bundle.crt
+      - name: reporting-operator-trusted-ca-bundle
+        configMap:
+          name: reporting-operator-trusted-ca-bundle
+          items:
+          - key: ca-bundle.crt
+            path: ca-bundle.crt
 {{- end }}
```

Comment:
What is this change?

Reply:
I wondered too, but I think it just brings indentation to correct?

Reply:
It was just misaligned and my editor was complaining about it. I called that out in one of the commits.